src/processor/operator/persistent/reader/csv/base_csv_reader.cpp

```diff
@@ -279,6 +279,11 @@ uint64_t BaseCSVReader::parseCSV(Driver& driver) {
     value_start:
         // state: value_start
         // this state parses the first character of a value
+        // skip the spaces after delimiter
+        if (buffer[position] == ' ') {
```

Comment:
I would suggest using `StringUtils::isSpace` instead.

Reply:
discussed with chang, this feature is against CSV standard, removed.